content/agent/faq/why-don-t-i-see-the-system-processes-open-file-descriptors-metric.md

```diff
@@ -3,7 +3,18 @@ title: Why don't I see the 'system.processes.open_file_descriptors' metric?
 kind: faq
 ---
 
-Users that enable the [process check][1] in a Datadog Agent running on Linux OSs notice that the `system.processes.open_file_descriptors` metric is not collected or reported by default. This occurs when the process being monitored by the process check runs under a different user than the Agent - the `dd-agent` user doesn't have full access to all files in `/proc`, which is where the Agent looks to collect data for this metric. A workaround for this can be achieved by running the Agent as `root`.
+Users that enable the [process check][1] in a Datadog Agent running on Linux OSs notice that the `system.processes.open_file_descriptors` metric is not collected or reported by default. This occurs when the process being monitored by the process check runs under a different user than the Agent - the `dd-agent` user doesn't have full access to all files in `/proc`, which is where the Agent looks to collect data for this metric.
+
+Users can enable the `try_sudo` option in the process check configuration and add the appropriate sudoers rules:
+```
+dd-agent ALL=NOPASSWD: /bin/ls /proc/*/fd/
+```
+This allows the process check to try using `sudo` to collect `system.processes.open_file_descriptors` metric.
```

Comment:
I'd be more specific about the rule we suggest, saying something on the line of "This allows the process check to use `sudo` to execute the `ls` command but only to list the contents of the path `/proc/*/fd/`"